woocommerce/woocommerce-android
# [Order Creation] Offer customer creation when email is entered but no customers found

Closes: #9840


### Description

The idea is to give a shortcut for the case when email is entered in the search but nothing is found -> we show a button that opens manual customer creation screen with email entered

### Testing instructions

Try that both on WCCore 8+ and below version
* Order -> Create order -> Put a valid email as the search param
* Notice a button -> click -> notice that email is prefilled

### Images/gif

<img width="388" alt="image" src="https://github.com/woocommerce/woocommerce-android/assets/4923871/10a7d487-894f-4e75-875f-3ef2cdb8d889">

https://github.com/woocommerce/woocommerce-android/assets/4923871/1487e354-bd29-4f31-b6ee-3bea6011ee3a
